supervisor/shared/serial.c

```diff
@@ -309,10 +309,6 @@ uint32_t serial_bytes_available(void) {
     }
     #endif
 
-    #if CIRCUITPY_USB_DEVICE
-    count += tud_cdc_available();
-    #endif
-
     // Board-specific serial input.
```

Comment:
I haven't dug into this so I don't know if there's any advantage to checking the USB device for input at this point, if you have and are confident it's of no value you can ignore me :grin:

Rather than dropping this could it be replaced with a bitwise OR
```count |= tud_cdc_available();```

Reply:
`count` is an integer, not a boolean, so the or'ing is not going to add the counts properly?

Reply:
Ah, yes, I was only thinking of the single character case, never mind...